src/main/kotlin/com/salesforce/zero/quickquartz/ExposedJobDetails.kt

```diff
@@ -28,51 +40,106 @@ object QuickQuartzJobDetails : Table("qrtz_job_details") {
 }
 
 /**
- * pojo
+ * poké (plain ol' kotlin entity)
  */
-data class JobDetail(
-    val schedName: String,
+data class JobDetailEntity(
+    val schedName: String = "QQ", // TODO just get rid of this from the schema
```

Comment:
It's the leading key of all the PKs, and it's a user-configurable value, so it would be possible to create multiple schedulers backed by the same database- I think of it as a namespace, essentially. It's cumbersome though because it's maintained as a global config value instead of a field on the (quartz) objects. We have jobGroup and triggerGroup available to use as scoped namespaces - e.g., tenant and job type.

Reply:
Ok, so, not useful to us, but could be useful for someone who wanted to use this library in some other scenario.